wallet/src/main/java/com/android/identity_credential/wallet/presentation/OpenID4VPPresentationActivity.kt

```diff
@@ -894,22 +894,48 @@ private class TimeChecks : JWTClaimsSetVerifier<SecurityContext> {
     }
 }
 
-private fun getFormat(clientMetadata: JsonObject): CredentialFormat {
-    // Search for these strings, in order.
+private fun getFormat(
+    clientMetadata: JsonObject,
+    presentationDefinition: JsonObject
+): CredentialFormat {
     val acceptedFormats = listOf(
```

Comment:
Add a comment that the order of the entries in the acceptedFormats list is important: we'll accept the first match (so we shouldn't convert this to a Map and risk losing the ordering information).

(otherwise I was tempted to recommend converting this to a map)  :)